src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -180,8 +198,10 @@ private static CultureInfo GetUserDefaultCulture()
 
         private static bool IsCustomCultureId(int cultureId)
```

Comment:
We may be able to move this into common code. However I will defer this until I add support for adding additional locale validation which will test "custom locales" which have a valid signature but don't exist and which will test this method to see if any changes are necessary.